src/sentry/conf/server.py

```diff
@@ -3146,12 +3146,16 @@ def custom_parameter_sort(parameter: dict) -> tuple[str, int]:
 
 SEER_ANOMALY_DETECTION_MODEL_VERSION = "v1"
 SEER_ANOMALY_DETECTION_URL = SEER_DEFAULT_URL  # for local development, these share a URL
-SEER_ANOMALY_DETECTION_TIMEOUT = 15
+SEER_ANOMALY_DETECTION_TIMEOUT = 5
```

Comment:
why are we going back to 5 seconds for the timeout?

Reply:
Ram resolved the timeouts and told me to roll it back. But I can take the change out of this PR.